data/World/Overworld.json

```diff
@@ -1630,7 +1857,8 @@
             "Sheik in Crater": "is_adult"
         },
         "exits": {
-            "DMC Central Local": "can_use(Goron_Tunic)"
+            "DMC Central Local": "can_use(Goron_Tunic)",
+            "DMC Pierre Platform": "(logic_fewer_tunic_requirements or can_use(Goron_Tunic)) and can_use(Distant_Scarecrow)"
```

Comment:
Going to the Pierre platform from central nearby is probably not safe. Worst case you might have to get there from the Fairy Fountain entrance, and I'm not going to say that's impossible, but it seemed extremely suspect. I recommend you instead exit to it from central local. And I think going from there should be safe even without FTR.

Reply:
Fixed. Please let me know if you think it's correct now.

Reply:
Yeah seems correct.